ClosedXML.Tests/Excel/Loading/LoadingTests.cs

```diff
@@ -142,6 +143,78 @@ public void CanLoadAndSaveFileWithMismatchingSheetIdAndRelId()
             }
         }
 
+        [Test]
+        [TestCase("PrintAreaTestCase1.xlsx")]
+        [TestCase("PrintAreaTestCase2.xlsx")]
+        [TestCase("PrintAreaTestCase3.xlsx")]
+        [TestCase("PrintAreaTestCase4.xlsx")]
+        [TestCase("PrintAreaTestCase5.xlsx")]
+        [TestCase("PrintAreaTestCase6.xlsx")]
+        [TestCase("PrintAreaTestCase7.xlsx")]
+        [TestCase("PrintAreaTestCase8.xlsx")]
+        [TestCase("PrintAreaTestCase9.xlsx")]
+        [TestCase("PrintAreaTestCase10.xlsx")]
+        [TestCase("PrintAreaTestCase11.xlsx")]
+        [TestCase("PrintAreaTestCase12.xlsx")]
+        [TestCase("PrintAreaTestCase13.xlsx")]
+        [TestCase("PrintAreaTestCase14.xlsx")]
+        public void CanLoadAndSavePrintArea(string file)
+        {
+            Assert.Multiple(() =>
```

Comment:
I don't see what this test checks. We can load and save file, but checks that it actually saves anything correctly, just that it saves something, possibly mangled. 

Also, I generally prefer not to use test files for exhaustive checks. It makes sense if you have a complex parsing during the load, but that is IMO wrong approach. It would be better to have 2 tests that they are loaded/savbed correctly (check that print areas are associated with correct sheet/whole workbook) and unit tests for XLPrintAreas that it can modify a formula (represented by `_xlnm.Print_Area` ) using API (e.g. adding another range to print areas would append range with a range union operator).

Reply:
The idea was to have a test that checks if the parsing and serialization of the ranges/formulas can be round-tripped without an exception. The other changes you proposed make that irrelevant so I will adapt that.

Reply:
your proposed changes make several test cases irrelevant

Reply:
I removed several test cases except one for loading/saving of the PrintArea